### PR TITLE
[SAT-174] Add NUM_LENGTH_PRESERVING alias format.

### PR DESCRIPTION
**Pull request by rozenbom**
## Description of changes in release / Impact of release:
Add NUM_LENGTH_PRESERVING alias format.

### Is this a breaking change?
  - [ ] Yes
  - [x] No

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
(insert text here)

### CODE REVIEW

File: satellite/db/migrations/versions/fe38630490f9_drop_enum_constaints.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# revision identifiers, used by Alembic.
+revision = 'fe38630490f9'
+down_revision = '71e95818e76e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_table(Alias.__tablename__)
+    op.drop_table(RuleEntry.__tablename__)
```
Comment (aslepakurov) [inline satellite/db/migrations/versions/fe38630490f9_drop_enum_constaints.py:23]: ???

Comment (rozenbom) [reply satellite/db/migrations/versions/fe38630490f9_drop_enum_constaints.py:23]: Well, sqlite and DB migrations are not always getting along together. That's why I disabled constraints creation for enum fields in this PR. Next changes in enum-fields will be smoother.

Comment (aslepakurov) [reply satellite/db/migrations/versions/fe38630490f9_drop_enum_constaints.py:23]: So this update is basically remove db tables w/constrainst and recreate w/o them?

Comment (rozenbom) [reply satellite/db/migrations/versions/fe38630490f9_drop_enum_constaints.py:23]: Correct. The tables will be created right after the migration is applied.

**Review by aslepakurov — APPROVED** [review]

**Pull request merged**